Python/ceval_macros.h

```diff
@@ -396,7 +396,7 @@ stack_pointer = _PyFrame_GetStackPointer(frame);
 #if TIER_TWO
 
 #define LOAD_IP() \
-do { ip_offset = (_Py_CODEUNIT *)_PyFrame_GetCode(frame)->co_code_adaptive; } while (0)
+do {} while (0)
```

Comment:
I'd be curious what @markshannon thinks of this change. It makes me think that the "abstractions" embodied by these `LOAD/STORE` macros are not as solid as we thought. And I wonder if the line that sets `prev_instr` should also be a macro. (Note that the `_SP` macros don't actually differ by tier.)

Reply:
This seems good to me. `LOAD_IP` should be a no-op in tier 2 execution. The IP should be virtual.

(In this context "virtual" means tracked by the region selector (`translate_bytecode_to_trace`) not at runtime)

Reply:
The `_SP` may not differ by tier, but the copy-and-patch compiler will need to treat them differently from the tier 2 interpreter.